app/components/explore/side-bar.js

```diff
@@ -150,6 +151,10 @@ export default Component.extend({
 
     toggleFilters() {
       this.set('showFilters', !this.showFilters);
+    },
+
+    toggleMap() {
+      this.toggleProperty('toggleMap');
```

Comment:
Don't call the property toggle map. Name it something that is relatable and easy-to-understand ..

`isMapVisible` or `isMapHidden` or something like that.

Reply:
done